controllers/runner_controller.go

```diff
@@ -134,13 +139,16 @@ func (r *RunnerReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 
 	ready := runnerPodReady(&pod)
 
-	if runner.Status.Phase != phase || runner.Status.Ready != ready {
+	if (runner.Status.Phase != phase || runner.Status.Ready != ready) && !r.ConfigureRunnersRBAC || runner.Status.Phase == "" && r.ConfigureRunnersRBAC {
 		if pod.Status.Phase == corev1.PodRunning {
 			// Seeing this message, you can expect the runner to become `Running` soon.
 			log.V(1).Info(
 				"Runner appears to have been registered and running.",
 				"podCreationTimestamp", pod.CreationTimestamp,
 			)
+			if r.ConfigureRunnersRBAC {
+				phase = "Created"
```

Comment:
Why do we need this treatment?

Reply:
not needed - I've removed in the next commits